### ISSUE TITLE
TehNut Maven repo not available

### ISSUE THREAD

**Issue by Muad3D** [TehNut-Mods/HWYLA#297]
#### Issue Description:
Gateway timeout during gradle build

#### What happens:
* What went wrong:
A problem occurred configuring root project 'DynamicTrees-1.14.4'.
> Could not resolve all files for configuration ':_compileJava_1'.
   > Could not resolve io.netty:netty-all:4.1.25.Final.
     Required by:
         project : > net.minecraft:client:1.14.4
      > Could not resolve io.netty:netty-all:4.1.25.Final.
         > Could not get resource 'https://maven.tehnut.info/io/netty/netty-all/4.1.25.Final/netty-all-4.1.25.Final.pom'.
            > Could not HEAD 'https://maven.tehnut.info/io/netty/netty-all/4.1.25.Final/netty-all-4.1.25.Final.pom'. Received status code 504 from server: Gateway Timeout

#### What you expected to happen:
Repo is available and can download packages

#### Steps to reproduce:
gradlew.bat build

____
#### Affected Versions (Do *not* use "latest"):

- Hwyla: 1.10.6-B67_1.14.4
- Minecraft: 1.14.4
- Forge: 28.2.23

**Comment by Muad3D** [comment]
Same issue happens on repeated attempts

**Comment by RRyyas** [comment]
Yeah, can't build the mod without this repo. Tried doing it manually by typing in the URL but it returns DNS_PROBE_FINISHED_NXDOMAIN. Any alternative links?

**Comment by mpfaff** [comment]
Not sure if it's related, but I get a 403 Forbidden error when connecting to the maven repo.

**Comment by Alatyami** [comment]
Getting 504 Gateway Timeout today.

**Comment by LemADEC** [comment]
Issue is still live
